### PR TITLE
Add '--format' for morphologyConverter app

### PR DESCRIPTION
**Pull request by mgeplf**
- Allows for choice of output format.  No change to the
  default at the moment (still h5v2), but allows for writing
  h5v1 files

### CODE REVIEW

**Comment by eile** [comment]
@mgeplf: We will merge this in about a week, since we need to build the 2015.R2 module right now.

**Comment by mgeplf** [comment]
Cool, no problem.  Let me know if the flag name isn't good, or any other changes that are needed.

File: apps/morphologyConverter.cpp [hunk 1]
```diff
@@ -75,11 +77,19 @@ int main( int argc, char* argv[] )
         return EXIT_FAILURE;
     }
 
+    brion::MorphologyVersion out_version = brion::MORPHOLOGY_VERSION_UNDEFINED;
+    if ( vm["format"].as< std::string >() == "h5v1" ) {
+        out_version = brion::MORPHOLOGY_VERSION_H5_1;
+    }
```
Comment (eile) [inline apps/morphologyConverter.cpp]: Style: no braces for one-liners, '{' on new-line for multi-liners.

Comment (eile) [inline apps/morphologyConverter.cpp]: style: CamelCapitalization. But I would just name the variable 'format'

File: apps/morphologyConverter.cpp [hunk 2]
```diff
@@ -115,7 +124,10 @@ int main( int argc, char* argv[] )
     readTime += clock.resetTimef();
 
     out.writeSectionTypes( *types );
-    out.writeApicals( *apicals );
+    if ( out.getVersion() != brion::MORPHOLOGY_VERSION_H5_1 ) {
+        out.writeApicals( *apicals );
+    }
```
Comment (eile) [inline apps/morphologyConverter.cpp]: as above.

**Comment by eile** [comment]
+1 otherwise

**Pull request merged**